TypeError: Cannot read property 'emit' of null

Heroku logs:
```
2018-10-18T13:01:22.605696+00:00 app[web.1]: /app/server.js:177
2018-10-18T13:01:22.605710+00:00 app[web.1]: serverState.activeController.emit("command", activeControllerState)
2018-10-18T13:01:22.605711+00:00 app[web.1]: ^
2018-10-18T13:01:22.605713+00:00 app[web.1]:
2018-10-18T13:01:22.605714+00:00 app[web.1]: TypeError: Cannot read property 'emit' of null
2018-10-18T13:01:22.605716+00:00 app[web.1]: at updateActiveController (/app/server.js:177:32)
2018-10-18T13:01:22.605717+00:00 app[web.1]: at Socket.display.on (/app/server.js:58:5)
2018-10-18T13:01:22.605718+00:00 app[web.1]: at Socket.emit (events.js:182:13)
2018-10-18T13:01:22.605719+00:00 app[web.1]: at /app/node_modules/socket.io/lib/socket.js:503:12
2018-10-18T13:01:22.605720+00:00 app[web.1]: at process._tickCallback (internal/process/next_tick.js:61:11)
2018-10-18T13:01:22.612746+00:00 app[web.1]: npm ERR! code ELIFECYCLE
2018-10-18T13:01:22.613093+00:00 app[web.1]: npm ERR! errno 1
2018-10-18T13:01:22.614110+00:00 app[web.1]: npm ERR! retris@1.0.0 start: `NODE_ENV=production node server.js -p $PORT`
2018-10-18T13:01:22.614211+00:00 app[web.1]: npm ERR! Exit status 1
2018-10-18T13:01:22.614395+00:00 app[web.1]: npm ERR!
2018-10-18T13:01:22.614543+00:00 app[web.1]: npm ERR! Failed at the retris@1.0.0 start script.
2018-10-18T13:01:22.614668+00:00 app[web.1]: npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
2018-10-18T13:01:22.621413+00:00 app[web.1]:
2018-10-18T13:01:22.621556+00:00 app[web.1]: npm ERR! A complete log of this run can be found in:
2018-10-18T13:01:22.621622+00:00 app[web.1]: npm ERR!     /app/.npm/_logs/2018-10-18T13_01_22_615Z-debug.log
2018-10-18T13:01:22.667471+00:00 heroku[web.1]: Process exited with status 1
2018-10-18T13:01:22.680655+00:00 heroku[web.1]: State changed from up to crashed
2018-10-18T13:01:22.683882+00:00 heroku[web.1]: State changed from crashed to starting
```

So the app tries to update the active controller and it doesn't exist.

--- a/server.js
+++ b/server.js
@@ -174,7 +174,9 @@ const updateDisplays = view => {
 const updateActiveController = view => {
   activeControllerState.activeView = view
   activeControllerState.queueLength = serverState.controllers.length
-  serverState.activeController.emit("command", activeControllerState)
+  if (serverState.activeController !== null) {
+    serverState.activeController.emit("command", activeControllerState)
+  }
 }
 
 // Updates the state of the passed controller

